sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -407,6 +407,16 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val REQUIRE_ALL_CLUSTER_KEYS_FOR_AGGREGATE =
```

Comment:
I picked up the similar config name with similar description in above config (`spark.sql.requireAllClusterKeysForCoPartition`) since the goal is very similar.